frontend/epfl-contact/controller.php

```diff
@@ -26,8 +26,8 @@ function epfl_contact_block($attributes, $inner_content) {
     ob_start();
 ?>
 
-    <div class="container <?php echo ($gray_wrapper) ? 'bg-gray-100 py-2 my-5' : 'my-3'; ?>">
-    <?php if ($gray_wrapper): ?><div class="bg-white p-4 p-md-5"><?php endif; ?>
+    <div class="container-full my-5 <?php if ($gray_wrapper) : ?>bg-gray-100 py-5<?php endif; ?>">
```

Comment:
Petit détail de finition:
l'espace doit être dans le if, sinon on se retrouve avec des classes qui ont des espaces sans raison
```suggestion
    <div class="container-full my-5<?php if ($gray_wrapper) : ?> bg-gray-100 py-5<?php endif; ?>">
```